### PR TITLE
fix: Add date value to HTML element

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
The element was empty, as item.date is undefined. The string interpolation created the empty string instead. With this page the value is filled.

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
Rebased on current main @joepavitt

**Comment by joepavitt** [comment]
Confused why this is a "fix" though, the published on dates show fine on the site as they are: https://flowforge.com/blog/2023/06/dashboard-announcement/

**Comment by ZJvandeWeg** [comment]
@joepavitt the value in the HTML property wasn't set. So humans indeed could read it, though machines found it hard to parse the output

**Review by joepavitt — APPROVED** [review]
Thanks for the clarification
